checkfiles.py

```diff
@@ -1002,10 +1002,8 @@ def fetch_files(session, url, search_query, out, include_unexpired_upload=False,
         r = session.get(item_url + '@@upload?datastore=database')
         if r.ok:
             upload_credentials = r.json()['@graph'][0]['upload_credentials']
-            job['upload_url'] = upload_credentials['upload_url']
             if fileObject.json()['s3_uri']:
```

Comment:
I know you didn't change this line, but this if statement is almost useless unless `s3_uri` can only be either an empty or non-empty string. If the s3 uri is not present, you'll get a KeyError. Could you confirm this is the correct behavior?

Reply:
I confirmed with @jnau that s3_uri will always be present for files, so this if statement is not needed.